modules/ROOT/pages/persistence-gateway.adoc

```diff
@@ -90,6 +90,18 @@ The output of this command should be similar to the following:
 192.168.2.101 - - [09/Apr/2021:16:35:55 +0000] "GET /api/v1/status/ready HTTP/1.1" 200 2 "" "kube-probe/1.18+"
 ----
 
+== Configure Authorize Namespace
```

Comment:
This section makes it seem like they need to configure authorized namespaces. This is not required. Only if they are already using authorized namespaces and they want to use persistence gateway, they need this step.